src/app/shared-component/dialog/dialog.component.ts

```diff
@@ -39,4 +40,22 @@ export class DialogComponent implements OnInit {
     this.onClick.emit(val);
     this.modalFadeIn = false;
   }
+
+  @HostListener('window:keydown', ['$event'])
+  onActionKeyPress(event: any): void {
+    let currentDefaultIndex = this.dialog.actionButtons.findIndex(i => i.default);
```

Comment:
Default is a new property to the dialog action data. Please refer to this file src/app/shared-component/dialog/dialog.ts. You should not get the error.